### PR TITLE
feat(cli): add interactive flag to cli params

### PR DESCRIPTION
**Pull request by BioPhoton**
This PR introduces the inactive flag. I refactored all ensure config files to use the flag.

Let me know if this helps and I will refactore other functions too. 
In this way we can chain a subset of ensure functions together to run e.g. history without the setup step.

How would this flag effect the setup task?

--

related to #114

### CODE REVIEW

File: packages/deprecation-crawler/src/lib/commands/default.command.ts [hunk 1]
```diff
@@ -24,19 +24,22 @@ export const defaultCommand: YargsCommandObject = {
             (config: CrawlConfig): CrawlerProcess =>
               askToSkip('Grouping?', group(config), {
                 precondition: async (r) =>
-                  r.deprecations?.some((d) => !d.group),
+                  r.deprecations?.some((d) => !d.group)
               }),
-            generateOutput,
-            updateRepository,
+            (config: CrawlConfig): CrawlerProcess =>
```
Comment (timdeschryver) [inline packages/deprecation-crawler/src/lib/commands/default.command.ts:29]: Imho it shouldn't be a choice, I think we should always format here?

**Review by timdeschryver — APPROVED** [review]
The default behavior is that the commands are run in interactive mode, right?
If that's the case this is definitely a 👍👍 for me.

File: packages/deprecation-crawler/src/lib/commands/default.command.ts (hunk 1 above)
Comment (BioPhoton) [reply packages/deprecation-crawler/src/lib/commands/default.command.ts:29]: Agree!

**Comment by timdeschryver** [comment]
@BioPhoton the tests should be fixed with the latest commit.
Can you give it a look, and if it's ok for you merge it in?

The commit also has a fix for #116.

**Pull request merged**